install-cockroachdb.md

```diff
@@ -178,7 +178,7 @@ $(document).ready(function(){
         <p>A C++ compiler that supports C++11. Note that GCC prior to 6.0 does not work due to <a href="https://gcc.gnu.org/bugzilla/show_bug.cgi?id=48891">this issue</a>. On Mac OS X, Xcode should suffice.</p>
       </li>
       <li>
-        <p>A <a href="http://golang.org/doc/code.html">Go environment</a> with a recent 64-bit version of the toolchain. The Makefile enforces the specific version required, as it is updated frequently. Be sure to set the <code class="highlighter-rouge">$GOPATH</code> and <code class="highlighter-rouge">$PATH</code> environment variables as described <a href="https://golang.org/doc/code.html#GOPATH">here</a>.</p>
+        <p>A <a href="http://golang.org/doc/code.html">Go environment</a> with a recent 64-bit version of the toolchain. The Makefile enforces the specific version required, as it is updated frequently.</p>
```

Comment:
I'm happy to put the specific version back in. That's what we had originally, until I changed the approach in this [commit]( https://github.com/cockroachdb/docs/commit/a22135dcaca0eb0116dc563a58c7ca9790621317) at @tamird's suggestion, I believe, and after having missed a version change.